docs/docs/configuration/object_detectors.md

```diff
@@ -309,14 +309,23 @@ RKNN support is provided using the `-rk` suffix for the docker image. Moreover,
 ### Configuration
 
 This `config.yml` shows all relevant options to configure the detector and explains them. All values shown are the default values (except for one). Lines that are required at least to use the detector are labeled as required, all other lines are optional.
+
 ```yaml
 detectors:                            # required
   rknn:                               # required
     type: rknn                        # required
+    # core mask for npu
+    core_mask: 0
+    # yolov8 model in rknn format to use; allowed calues: n, s, m, l, x
+    yolov8_rknn_model: n
+    # minimal confidence for detection
+    min_score: 0.5
```

Comment:
I don't think this should be configurable, it is not configurable for other models